Copyright (C) 2009  Volker Grabsch

Permission is hereby granted, free of charge, to any person obtaining
a copy of this software and associated documentation files (the
"Software"), to deal in the Software without restriction, including
without limitation the rights to use, copy, modify, merge, publish,
distribute, sublicense, and/or sell copies of the Software, and to
permit persons to whom the Software is furnished to do so, subject
to the following conditions:

The above copyright notice and this permission notice shall be
included in all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

diff -ru proj-4.7.0.orig/src/pj_mutex.c proj-4.7.0/src/pj_mutex.c
--- proj-4.7.0.orig/src/pj_mutex.c	2009-09-23 21:03:26.000000000 +0200
+++ proj-4.7.0/src/pj_mutex.c	2009-10-03 23:32:03.000000000 +0200
@@ -37,7 +37,7 @@
 #include <proj_api.h>
 #endif
 
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(MUTEX_pthread)
 #  define MUTEX_win32
 #endif
 
